imaginary.go

```diff
@@ -268,11 +268,17 @@ func checkHTTPCacheTTL(ttl int) {
 	}
 }
 
-func parseCustomHeaders(headers string) []string {
-	if headers == "" {
-		return []string{}
+func parseForwardHeaders(forwardHeaders string) []string {
+	var headers []string
+	if forwardHeaders == "" {
+		return headers
 	}
-	return strings.Split(headers, ",")
+	for _, header := range strings.Split(forwardHeaders, ",") {
```

Comment:
This is a great improvement over your previous implementation. However there is still a gap that allows for miss-configuration, namely if the flag has a space between commas. Which might not be that far fetched if you take newline characters into account: https://play.golang.org/p/WUh_upVSy2F

So if you trim white-space before the empty check, I think we've nailed it 💪